### PR TITLE
Feature/submission bughunt fixes

### PR DESCRIPTION
**Pull request by vikasrohit**
@parthshah @fnisen

### CODE REVIEW

File: app/directives/tc-fp-file-input/tc-fp-file-input.directive.js [hunk 1]
```diff
@@ -32,12 +32,15 @@ import _ from 'lodash'
           scope.filePath += _.join([userId, scope.fieldId, (new Date()).valueOf()], '-') + '.zip'
         }
         var dragAreaClasses = 'tc-fp-file-drag-drop'
-        // set extensions
+        var dragText = ''
+        // set mimeTypes
         if (scope.fieldId.indexOf('ZIP') > -1) {
-          scope.extensions = '.zip'
+          scope.mimeTypes = 'application/zip'
```
Comment (parthshah) [inline app/directives/tc-fp-file-input/tc-fp-file-input.directive.js]: @vikasrohit we've had lots of problems with mimetypes with our current implementation. So I wanted to switch to extensions only.

Comment (vikasrohit) [inline app/directives/tc-fp-file-input/tc-fp-file-input.directive.js]: Okay, so should I revert the changes?
